src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -261,7 +255,8 @@ trait AttributeComponent {
               value_number double DEFAULT NULL,
               value_boolean bit(1) DEFAULT NULL,
               value_entity_ref bigint(20) unsigned DEFAULT NULL,
-              list_index int(11) DEFAULT NULL, """
+              list_index int(11) DEFAULT NULL,
+              list_length int(11) DEFAULT NULL, """
```

Comment:
I am not following. The column should be nullable and the default value should be null which is how it is if a default is not specified. What is the argument for declaring DEFAULT NULL? Also the default value will almost never be used.